File: Vorbereitung/Übungen/2/2-2.c
int main() {
    int x;
    scanf("%d", &x);

    switch (x)
    {
    case 90 ... 100:
        printf("Sehr gut");
        break;
    case 75 ... 89: 
        printf("Gut");
        break;
    case 60 ... 74:
        printf("Befriedigend");
        break;
    case 50 ... 59:
        print("Ausreichend");
    default:
        printf("Nicht bestanden");
        break;
    }
    return 0;
}

File: Vorbereitung/Übungen/2/2-2.cs
namespace Uebung;

class Program {
    static void Main(string [] Args) {
        int x = Console.ReadLine();

        string Note = x switch {
            int p when p <= 100 && p >= 90 => "Sehr gut",
            int p when p < 90 && p >= 75 => "Gut",
            int p when p < 75 && p >= 60 => "Befriedigend",
            int p when p < 60 && p >= 50 => "Ausreichend"
            _ => "Nicht bestanden"
        };

        print(Note);
    }
}